### PR TITLE
Fix minor formatting issues

### PR DESCRIPTION
**Pull request by hovaesco**
Fix minor formatting issues.

### CODE REVIEW

File: content/en/references/coverage/index.md [hunk 1]
```diff
@@ -113,20 +113,20 @@ The table is a work in progress and will be updated as more functions are implem
 |BUILD_SCOPED_FILE_URL|❓|❓|
 |BUILD_STAGE_FILE_URL|❓|❓|
 |CASE|❓|❓|
-|CAST , ::|❓|❓|
+|CAST|❓|❓|
```
Comment (eruditmorina) [inline content/en/references/coverage/index.md:116]: I think `, ::` is left to suggest `::` as the other alternative to `CAST`:
https://docs.snowflake.com/en/sql-reference/functions/cast

But I may agree to put it as a code format to look better maybe?

File: content/en/references/coverage/index.md [hunk 2]
```diff
@@ -113,20 +113,20 @@ The table is a work in progress and will be updated as more functions are implem
...
-|CHR , CHAR|❓|❓|
+|CHR, CHAR|❓|❓|
 |COALESCE|✅|✅|
 |COLLATE|❓|❓|
 |COLLATION|❓|❓|
 |COMPLETE (SNOWFLAKE.CORTEX)|❓|❓|
 |COMPLETE_TASK_GRAPHS|❓|❓|
 |COMPRESS|❓|❓|
-|CONCAT , |||✅|✅|
+|CONCAT|✅|✅|
```
Comment (eruditmorina) [inline content/en/references/coverage/index.md:129]: Same here I would think `, ||` is to suggest `||` as an alternative syntax for `CONCAT`:
https://docs.snowflake.com/en/sql-reference/functions/concat

**Comment by eruditmorina** [comment]
I can also see that there are some functions that we support by now that may need to be updated here such as:

`AS_CHAR , AS_VARCHAR, AS_DECIMAL , AS_NUMBER, AS_DOUBLE , AS_REAL, SUM, MODE, ANY_VALUE `

cc: @HarshCasper

**Review by whummer — COMMENTED** [review]
Kudos for proactively fixing the docs in here @hovaesco ! 🚀 Unfortunately, the `index.md` file is auto-generated, so these changes would be overwritten by the next CI run (we currently update the docs on a weekly schedule). So, we'd need to make those changes at the source directly - to dive into that in more detail..! 👍

**Comment by hovaesco** [comment]
Thanks @whummer it totally make sense, good insight 👍  I'm closing the PR then. Will fix in the code where applicable.

**Pull request closed without merging**